tests/test_diffusers.py

```diff
@@ -1776,19 +1776,21 @@ def test_train_text_to_image_sdxl(self):
                  --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0
                  --pretrained_vae_model_name_or_path stabilityai/sdxl-vae
                  --dataset_name lambdalabs/pokemon-blip-captions
```

Comment:
@regisss : 
All tests that use `--dataset_name lambdalabs/pokemon-blip-captions` probably should bedisabled. The dataset has been taken down

We need to figure out a different dataset for the next release

What's the best way to disable these tests?

Reply:
Yes, I saw that too. Not sure exactly what the best way of disabling them is, I'll do it in another :+1: